index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"/>
+</head>
+<body>
+  <div class="header">
+    <a href="#" class="welcome miss-desktop">Welcome</a>
```

Comment:
- [ ] Excellent job using `a` for the logo 🥇 However, while implementing the change, please make sure that your design follows the fgma template. I noticed that because of using `a` you have a `text-decoration` which is not present in figma template ⚒ Please make sure that you use `text-decoration: none` for `a` so that we have a design close to figma template as much as possible 🎯

### _Your design previously 🔽_
![image](https://user-images.githubusercontent.com/106119105/219649236-e20aff84-ee56-471c-9ffd-4759799f1fc7.png)

### _Your design now 🔽_
![image](https://user-images.githubusercontent.com/106119105/219649284-f228b429-ace6-4b97-a88c-b5783a0a6db9.png)

### _Figma template 🔽_
![image](https://user-images.githubusercontent.com/106119105/219649510-53724ed3-109e-46cc-9080-fd82767011e3.png)